ToricVarieties/makefile

```diff
@@ -12,7 +12,8 @@ docclean:
 	(cd doc ; ./clean)
 
 test:	doc
-	gap maketest.g
+	(cd tst ; ./clean)
```

Comment:
This deletes the tests right before they are executed. Probably you should move this line right before the call to `gap makedoc.g` in the target `doc/manual.six`.

Reply:
Done in https://github.com/homalg-project/ToricVarieties_project/pull/49/commits/5c5d4ad71fe18bc321a45a9669c6d02640b5d836.